tests/inspectdb/tests.py

```diff
@@ -310,16 +310,19 @@ def test_include_views(self):
             with connection.cursor() as cursor:
                 cursor.execute('DROP VIEW inspectdb_people_view')
 
-    @skipUnless(connection.vendor == 'postgresql', 'PostgreSQL specific SQL')
+    @skipUnlessDBFeature('supports_materialized_views')
     def test_include_materialized_views(self):
-        """inspectdb --include-views creates models for database materialized views."""
+        """
+        inspectdb --include-views creates models for database materialized
+        views.
+        """
         with connection.cursor() as cursor:
             cursor.execute(
-                'CREATE MATERIALIZED VIEW inspectdb_people_materialized_view AS '
+                'CREATE MATERIALIZED VIEW inspectdb_people_materialized AS '
```

Comment:
I changed the name of materialized view to fit in 30 characters limit in Oracle.